feat: add quickstart example for web (#3)
ETOSPHERES-Labs/etopay-sdk-quickstart-ts: README.md

## Installation

The ETOPay SDK is available on NPM, making it easy to integrate into your project. You can find it here:  

* Node - [ETOPay SDK Node](https://www.npmjs.com/package/@etospheres/etopay-sdk-wasm-node).
* Web - [ETOPay SDK Web](https://www.npmjs.com/package/@etospheres/etopay-sdk-wasm-web).

To install it as a dependency, simply run the following command in your project directory: 

```bash
bun install @etospheres/etopay-sdk-wasm-node
```
or

```bash
bun install @etospheres/etopay-sdk-wasm-web
```

> Currently, we only support `bun` for running the SDK, as `node` does not work correctly with the current version. We recommend using `bun` to ensure proper functionality. This restriction is only for the node-example.

This will add the ETOPay SDK to your project, and you'll be ready to start using it (as it is already done in these examples) ! 😊

## Getting started

- Copy the .env.example file to .env and set the missing values
- Go to `https://etopayapp.etospheres.com` and get the SDK configuration for you project

> - Set the SDK configuration in `src/index.ts`

Reviewer: maybe we should split getting started into two sections. For web-example you have to edit Sdk config in `etopay-sdk-quickstart-ts/web-example/src/pages/index.js`

Author: Yeah sure, I can do it. I thought there were not many differences between the node and web examples but apparently there are some 😅

The ETOPay SDK is available on NPM, making it easy to integrate into your project. You can find it here:  

* Node - [ETOPay SDK Node](https://www.npmjs.com/package/@etospheres/etopay-sdk-wasm-node).
* Web - [ETOPay SDK Web](https://www.npmjs.com/package/@etospheres/etopay-sdk-wasm-web).

To install it as a dependency, simply run the following command in your project directory: 

```bash
bun install @etospheres/etopay-sdk-wasm-node
```
or

```bash
bun install @etospheres/etopay-sdk-wasm-web
```

> Currently, we only support `bun` for running the SDK, as `node` does not work correctly with the current version. We recommend using `bun` to ensure proper functionality. This restriction is only for the node-example.

This will add the ETOPay SDK to your project, and you'll be ready to start using it (as it is already done in these examples) ! 😊

## Getting started

- Copy the .env.example file to .env and set the missing values
- Go to `https://etopayapp.etospheres.com` and get the SDK configuration for you project
- Set the SDK configuration in `src/index.ts`
- Run the node example with `cd node-example && bun src/index.ts`
- Run the web example with `cd web-example && pnpm run dev`

> !!! Note

Reviewer 2: This does actually not give you a note in GitHub. Better to use the standard markdown note: `>`

> To run the web example locally, you need to enable _CORS_ for your Keycloak setup. Therefore, go to the **Clients** section in the Keycloak Admin Console and add `http://localhost:8080` to the **Web Origins** field. Then, ensure the **Standard Flow** is enabled and _Direct Access Grants_ is disabled for the client. Save your changes, and this will allow your app to communicate with Keycloak properly while using the recommended authorization code flow.

Reviewer 2: This is actually not correct, in this example we are still using the direct Access Grants flow.... so the note here is misleading.
```suggestion
To run the web example locally, you need to enable _CORS_ for your Keycloak setup. Therefore, go to the **Clients** section in the Keycloak Admin Console, select your client and add `http://localhost:8080` to the **Web Origins** field in the **Settings** tab. Save your changes, and this will allow your app to communicate with Keycloak properly while using the (not) recommended direct access grant flow.
```
